react/components/ProductMessagesExport.tsx

```diff
@@ -46,6 +58,10 @@ const {
     defaultMessage: '',
     id: 'admin/messages.required',
   },
+  somethingWrongMessage: {
+    defaultMessage: '',
+    id: 'admin/messages.step1.something-wrong-toast',
```

Comment:
non-blocking: internationalization keys ideally doesn't state where it goes. Right now it is displayed in a toast, but that could be used in another component.